cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -100,9 +101,12 @@ public void setUp() throws Exception {
 
     boolean deleteUponTeardown = false;
     if (!getNamespaceClient().exists(configuredNamespace.toId())) {
-      getNamespaceClient().create(new NamespaceMeta.Builder().setName(configuredNamespace.toId()).build());
-      // if we created the configured namespace, delete it upon teardown
-      deleteUponTeardown = true;
+      // Do not create a new namespace if its a default namespace.
```

Comment:
the changes on line 104-109 can be removed since the changes on lines 160+ handle it.